gammapy/irf/effective_area_table.py

```diff
@@ -444,6 +446,67 @@ def read(filename):
         hdu_list = fits.open(filename)
         return EffectiveAreaTable2D.from_fits(hdu_list)
 
+    def to_effectiveareatable(self, offset, energ_lo = None, energ_hi = None):
+        """Evaluate at a given offset and return effective area table object.
+
+        The energy thresholds in the effective area table object are not set.
+        If the effective area table is intended to be used for spectral analysis,
+        the final energy binning should be given at this point, since the
+        effective area table class is not able to perform interpolation at
+        this point.
+
+        Parameters
+        ----------
+        offset : `~astropy.coordinates.Angle`
+            offset
+        energy : `~astropy.units.Quantity`
+            energy
+
+        Returns
+        -------
+        eff_area_table : `EffectiveAreaTable`
+             Effective area table class. 
```

Comment:
Remove "class".
Python is very dynamic and you can return a class.
But here you return an instance, which is pretty much what we do all the time, so no need to add "object" or "instance" at the end of this line.